### DISCUSSION TITLE
How to do both analysis solo and trio for the same sample ?

### DISCUSSION THREAD

**Discussion by bnouyou** [broadinstitute/seqr#2003] [Q&A]
Hello,
I have a question about the utilisation of seqr for one case. I have loaded a sample in "single" analysis. Then, I would like to analyse the family of this sample, but this time, this sample will not be the proband, it will be the mother of the proband.
How can I do for doing the trio analysis without deleting the solo analysis ? Because I will use the same VCF file for the mother.

Thank you in advance for your help.

**Comment by hanars** [comment]
so you should load both the proband and the mother into a new index and add that index to the existing project. It won't erase any of the analysis you have already done

**Reply by bnouyou** [reply]
But what if the mother has the same individual_ID in both analysis ? I mean, when she is the proband (in single analysis) and when she  is not the proband (in trio analysis). In the first case, I have created a family in which she is alone. And then, she will be included in another family for the second analysis. Should I change the individual_ID ? Because, I will not sequenced the mother a second time, I will use the same data for her, so I will have the same individual_ID.

I don't know if my question makes sense.. Thank you for your help.

**Reply by hanars** [reply]
Your question makes sense now, but this is not a use case seqr handles - we do not have a concept of a single family where we would ever want to do analysis without all possible available data. Why are you looking for a "solve" in the mother that isn't also the "solve" for her child? In general, if we have a trio we never do "single analysis" ever again. 

One thing you can try is in the "inheritance" panel for search you can toggle the affected status for an individual. So you can set the "affected" status of the mother to "affected" and the proband to "unknown"  for that particular search and it will return results as if you only care about the mother's affected status which is basically doing a single analysis for her

**Reply by bnouyou** [reply]
This is a particular case actually. We have analyzed the mother alone because she has a pathology and we didn't have her parents. So we have analyzed her as proband. And now, we have received a sample of her fetus, but as I understood, it should be analyzed for another pathology. And for this fetus we have the mother (we already had her sample) and the father, for the complete family analysis. So I will create the family for this trio and this time, the fetus will be the proband. The biologists I work with asked me if we could do the trio analysis without deleting the "single" analysis of the mother.

We sometimes had one patient "alone" for analysis and then we get the family, so we can reanalyse the entire family as trio, but in that cases the proband stayed the same, so we didn't have problems. That's why I was asking the question here.

Thank you for your suggestion about the "Heritage" panel and changing status of the individuals. I think that the biologists want is to keep the possibility to reanalyse the mother as proband.
